app/views/cards/_form.html.erb

```diff
@@ -1,7 +1,8 @@
-<%= simple_form_for(@card, html: { class: 'form-group' }) do |f| %>
+<%= simple_form_for( @card, html: { class: 'form-group' }) do |f| %>
   <%= f.error_notification %>
   <%= f.input :original_text, label: 'Оригинальный текст' %>
   <%= f.input :translated_text, label: 'Перевод текста' %>
+  <%= f.input :user_id, as: :hidden, input_html: {value: "#{current_user.id}"} %>
```

Comment:
maybe you will add this to params inside controller? Becuase in this case, if you as a user with id=2 change in console your id to 3, you will create card for foreign user. Looks like security issue.